portfolio/src/main/webapp/index.html

```diff
@@ -42,74 +40,93 @@ <h5 class="text-white h4">Menu</h5><br>
+        </a> 
+      </div>
+    </div> 
+    <div class="row peach">
+      <div class="col center-text">
+        Click on these buttons for random messages 
+      </div>
     </div>
-  </div>
-
-  <div class="row peach">
-    <div class="col-3">
+    <div class="row peach padding-bottom">
+      <div class="col-3">
+      </div>
+      <div class="col-3 right-text">
+        <button class="btn btn-outline-primary" onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+      </div>
+      <div class="col-3 left-text">
+        <button class="btn btn-outline-primary" onclick="addRandomFunFact()">Fun Fact</button>
+        <div id="fact-container"></div> 
+      </div>
     </div>
-    <div class="col-3 right-text">
-      <button class="btn btn-outline-primary" onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="row peach">
+      <div class="col-2">
+      </div>
+      <div class="col-8">
+        <hr>
+      </div>
     </div>
-    <div class="col-3 left-text">
-      <button class="btn btn-outline-primary" onclick="addRandomFunFact()">Fun Fact <br></button>
-      <div id="fact-container"></div> 
+    <div class="row peach">
+      <div class="col-3">
+      </div>
+      <div class="col-6 center-text margin-bottom">
+        <h1>Comments</h1>
+        Enter a comment!
+      </div>
     </div>
-    <br><br><br>
-  </div>
-
-  <div class="row peach">
-	<div class="col center-text">
-      <button class="btn btn-outline-primary" onclick="getHello()">Fetch the hello!</button>
-      <div id="hello-container"></div>
-	</div>
-  </div>
-
-  <div class="row bg-dark text-light footer">
-    <div class="col center-text">
-      Contact<br><br>chrismohri@google.com<br>ccm244@cornell.edu<br><br>
+    <div class="row peach">
+      <div class="col-3">
+      </div>
+      <div class="col-6 center-text margin-bottom">
+        <form action="/data" method="POST">
+          <textarea name="text-input" class="form-control" id="exampleFormControlTextarea1" rows="3">comment...</textarea>
+          <input type="submit" class="btn btn-outline-primary" value="Submit comment" style="margin-top: 20px; margin-bottom:30px;"/>
+          <hr>
+          <div class="form-group">
+            <label for="exampleFormControlSelect1">Select nummer of visible comments</label>
+            <select class="form-control" id="exampleFormControlSelect1" onchange="getCommentSection()">
+              <option>5</option>
+              <option>10</option>
+              <option>15</option>
+            </select>
+          </div>
+        </form>
+        <input type="submit" class="btn btn-outline-primary" value="Delete all comments" onclick="deleteComments()" style="margin-bottom: 40px"/>				
+        <ul id="history"></ul>
+      </div>
     </div>
+    <div class="row bg-dark text-light footer">
+      <div class="col center-text padding-bottom">
+        Contact<br><br>
```

Comment:
nit: You could optionally remove the `<br>`s, wrap the "Contact" in a `<div>`, give it a class, and apply some margin-bottom. Not a big deal though.

Reply:
fixing in the subbranch